pkg/cmd/setup.go

```diff
@@ -237,6 +250,33 @@ func valueFromEnvironmentNotice(name string) string {
 	return fmt.Sprintf("Using %s from the environment. To disable this, use --no-read-env.", name)
 }
 
+// we're looking for duplicate paths and more than one repo being defined without a path.
+func setupFileErrorCheck(repos []models.ProjectConfig) {
+	// check to see if a repo isn't specifying a path and more than one repo exists
+	pathCount := make(map[string]int)
```

Comment:
Should we resolve the absolute path before performing this check?

Reply:
Hm.. Not sure. In theory that shouldn't really matter in the end (duplicates would resolve in the same way). If we did resolve them first and then printed the path in the error, that path would also no longer actually match what they had in the `doppler.yaml` file, so it might even end up being confusing (e.g., `.` becomes `/home/foo/something`).